doc/release-notes/rl-2511.section.md

```diff
@@ -16,6 +16,16 @@
   If a newer C++ library feature is not available on the default deployment target, you will need to increase the deployment target.
   See the Darwin platform documentation for more details.
 
+- A great number of changes to CUDA packaging:
+  - CUDA 13 is now available.
+  - Creation of CUDA package sets has been rewritten to remove module system evaluation.
+  - More packages provided by NVIDIA as binary archives (also called redistributables) are now available.
+  - Packages which fail to evaluate and depend on CUDA packages can produce helpful diagnostic messages during evaluation when `--show-trace --trace-verbose` are used.
+  - Package expressions are being updated to support building against older releases (where appropriate) to support out-of-tree consumers like [cuda-legacy](https://github.com/nixos-cuda/cuda-legacy).
+  - `_cuda.lib.licenses` was created to introduce several NVIDIA licenses new to Nixpkgs and used almost entirely within the CUDA package set.
+  - `_cuda.lib.getRedistSystem` had an API change to accommodate a new dependency on CUDA version.
```

Comment:
Disclaimer about internals&subject to change

Reply:
I don't see the disclaimers in the manual either, and I'd like to see them here as well anyway